src/client/api/json_parser/outputs/output_alias.c

```diff
@@ -69,9 +71,11 @@ int json_output_alias_deserialize(cJSON *output_obj, output_alias_t **alias) {
   }
 
   // stateMetadata
-  if (json_get_byte_buf_str(output_obj, JSON_KEY_STATE_METADATA, state_metadata) != JSON_OK) {
-    printf("[%s:%d]: getting %s json byte buffer failed\n", __func__, __LINE__, JSON_KEY_STATE_METADATA);
-    goto end;
+  if (cJSON_GetObjectItemCaseSensitive(output_obj, JSON_KEY_STATE_METADATA) != NULL) {
+    if (json_get_byte_buf_str(output_obj, JSON_KEY_STATE_METADATA, state_metadata) != JSON_OK) {
+      printf("[%s:%d]: getting %s json byte buffer failed\n", __func__, __LINE__, JSON_KEY_STATE_METADATA);
+      goto end;
+    }
   }
```

Comment:
State metadata is omitted in JSON response if it is NULL.
`json_get_byte_buf_str` returns `JSON_KEY_NOT_FOUND` if there is no state metadata.
First if statement is needed otherwise second if statement would need to be extended to check also for `JSON_KEY_NOT_FOUND`.